backend/tasks/user.py

```diff
@@ -7,58 +7,90 @@
 import json
 import logging
 from flask_login import login_required
-from data.model import (user, db_transaction)
+from data.model import user, db_transaction
 from data.database import (
     Repository,
     RepositoryBuild,
     RepositoryBuildTrigger,
     RepoMirrorConfig,
 )
 from data.queue import WorkQueue
+from utils import create_transaction as tf
+
 logger = logging.getLogger(__name__)
 
 
-class UserTask(Resource):
+NOTIFICATION_QUEUE_NAME = "notification"
+DOCKERFILE_BUILD_QUEUE_NAME = "dockerfilebuild"
+REPLICATION_QUEUE_NAME = "imagestoragereplication"
+CHUNK_CLEANUP_QUEUE_NAME = "chunk_cleanup"
+NAMESPACE_GC_QUEUE_NAME = "namespacegc"
+REPOSITORY_GC_QUEUE_NAME = "repositorygc"
+EXPORT_ACTION_LOGS_QUEUE_NAME = "exportactionlogs"
+SECSCAN_V4_NOTIFICATION_QUEUE_NAME = "secscanv4"
+
+
+image_replication_queue = WorkQueue(REPLICATION_QUEUE_NAME, tf, has_namespace=False)
+dockerfile_build_queue = WorkQueue(DOCKERFILE_BUILD_QUEUE_NAME, tf, has_namespace=True)
+notification_queue = WorkQueue(NOTIFICATION_QUEUE_NAME, tf, has_namespace=True)
+secscan_notification_queue = WorkQueue(SECSCAN_V4_NOTIFICATION_QUEUE_NAME, tf, has_namespace=False)
+export_action_logs_queue = WorkQueue(EXPORT_ACTION_LOGS_QUEUE_NAME, tf, has_namespace=True)
+repository_gc_queue = WorkQueue(REPOSITORY_GC_QUEUE_NAME, tf, has_namespace=True)
+namespace_gc_queue = WorkQueue(NAMESPACE_GC_QUEUE_NAME, tf, has_namespace=False)
```

Comment:
Some of these references aren't being used, is there a reason they're included?

Reply:
Related to the other comment.